targets/TARGET_TOSHIBA/TARGET_TMPM066/serial_api.c

```diff
@@ -239,3 +239,11 @@ void serial_pinout_tx(PinName tx)
 {
     pinmap_pinout(tx, PinMap_UART_TX);
 }
+
+void serial_break_set(serial_t *obj)
```

Comment:
@0xc0170 Line control register is not available in M066 controller. Due to this these 2 functions are not implemented and hence they are kept empty.